x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/license/LicensingTests.java

```diff
@@ -190,6 +202,82 @@ public void testNodeJoinWithoutSecurityExplicitlyEnabled() throws Exception {
         }
     }
 
+    public void testWarningHeader() throws Exception {
+        Request request = new Request("GET", "/_security/user");
+        RequestOptions.Builder options = request.getOptions().toBuilder();
+        options.addHeader("Authorization", basicAuthHeaderValue(SecuritySettingsSource.TEST_USER_NAME,
+            new SecureString(SecuritySettingsSourceField.TEST_PASSWORD.toCharArray())));
+        request.setOptions(options);
+
```

Comment:
nit: Do we need all the empty lines?

Reply:
Yes, it is intentional (see Tim's comment above). If Authentication fails it does make sense to not reveal any additional information